plugins/woocommerce/includes/admin/meta-boxes/views/html-product-data-variations.php

```diff
@@ -150,3 +150,25 @@
 		<?php endif; ?>
 	</div>
 </div>
+<script type="text/template" id="tmpl-wc-modal-set-price-variations">
+	<div class="wc-backbone-modal">
+		<div class="wc-backbone-modal-content">
+			<div class="components-modal__content woocommerce-set-price-variations" role="document">
+				<div class="components-modal__header">
+					<h2><?php echo esc_attr( $modal_title ); ?></h2>
+				</div>
+				<div class="woocommerce-usage-modal__wrapper">
+					<div class="woocommerce-usage-modal__message">
+						<span>Add price to all variations that don't have a price (<?php echo esc_attr( get_woocommerce_currency_symbol() ); ?> <?php echo esc_textarea( get_woocommerce_currency() ); ?>)</span>
```

Comment:
This should be surround with the `esc_html_e` right?

Reply:
Nice catch! I add it in commit [6af604f](https://github.com/woocommerce/woocommerce/pull/34948/commits/6af604f32b7cfab898d972502e24f4acc1b1bb5a)